Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Footer.css';
+
+
+let heart = 0;
+
+// function clickTest () {
+//     reload += 1;
+//     console.log('yup');
+//     return reload;
+// }
+
+
+const Footer = (props) => {
+    let reload = 0;
+    function clickTest () {
+        reload += 1;
+        console.log('yup');
+        return reload;
+    };
+    
+    return (
+        <div className="icons">
+            <i class="far fa-comment"></i>
+            <i class="fas fa-sync-alt" onClick={clickTest}> {reload}</i>
```

Comment:
If you have the function outside of your functional component you just have to mess with binding. `onClick = { () => this.clickTest() }` or `onClick = { this.clickTest() }`